### ISSUE TITLE
ignoring unrecognized descriptive metadata

### ISSUE THREAD

**Issue by dauwhe** [w3c/epub-specs#1475]
https://w3c.github.io/epub-specs/epub33/rs/#manifest says: 

> Reading Systems MAY optimize the rendering depending on the properties set in the properties attribute (e.g., disable a rendering process or use a fallback). Reading Systems MUST ignore all descriptive metadata properties that they do not recognize.

I take this to mean that if a reading system finds an unknown value in the `properties` attribute of the `item` element, it shouldn't fail (although this is an EPUBCheck error). 

Possible test: 

```xml
<item properties="foo bar" href="index.xthml" media-type="application/xml+html"/> 
```

So we're saying that the reading system should [1] display the file as usual and [2] not warn the reader about anything?

**Comment by mattgarrish** [comment]
> So we're saying that the reading system should [1] display the file as usual and [2] not warn the reader about anything?

Right, they're descriptive info for the reading system, so if they aren't recognized there's nothing to warn the user about.

**Comment by dauwhe** [comment]
Should we rewrite to make it a little more specific? 

> Reading Systems MUST ignore values of the `properties` attribute that they do not recognize.

Elsewhere we have

> Reading Systems SHOULD ignore all meta elements whose property attributes define expressions they do not recognize. A Reading System MUST NOT fail when encountering unknown expressions.

(and now I wonder why that's a SHOULD)

**Comment by mattgarrish** [comment]
> Should we rewrite to make it a little more specific?

Ya, it sounds weirdly like `meta` property values might somehow be in scope right now.


> > Reading Systems SHOULD ignore all meta elements whose property attributes define expressions they do not recognize
> 
> and now I wonder why that's a SHOULD

I think this was just for flexibility in presentation. Even though a reading system might not know a particular metadata field, it could still present it to users (e.g., using the property as the label in a list of all metadata, where it might translate the known fields to something more human readable). We don't want to exclude reading systems offering up information.

**Comment by mattgarrish** [comment]
Can this issue be closed with #1476 leaving the "should" alone?
